src/main/java/ChatBox.java

```diff
@@ -0,0 +1,123 @@
+package packagedirectory.test;
+
+import packagedirectory.test.DukeException;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+import java.util.Scanner;
+
+public class ChatBox {
```

Comment:
Interesting use of a custom ChatBox class! Something I have not seen so far. Great job!